src/Core/Settings.cpp

```diff
@@ -3612,6 +3612,9 @@ If enabled, functions 'least' and 'greatest' return NULL if one of their argumen
 )", 0) \
     DECLARE(Bool, h3togeo_lon_lat_result_order, false, R"(
 Function 'h3ToGeo' returns (lon, lat) if true, otherwise (lat, lon).
+)", 0) \
+    DECLARE(Bool, geotoh3_lon_lat_input_order, false, R"(
```

Comment:
Need to add new setting to `SettingsChangesHistory.cpp`. Related [build failure](https://s3.amazonaws.com/clickhouse-test-reports/json.html?PR=78852&sha=99fc98d6b4e3dfa038ac318779f6c3dbe92dea72&name_0=PR&name_1=Fast+test&name_2=Tests).